petab/visualize/plot_data_and_simulation.py

```diff
@@ -8,6 +8,8 @@
 import petab
 
 from typing import Union, Optional, List
+import matplotlib.pyplot as plt
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Unable to import 'matplotlib.pyplot'](https://app.codacy.com/manual/dweindl/PEtab/pullRequest?prid=4827411)